cmd/guardian/main.go

```diff
@@ -123,7 +123,7 @@ func main() {
 	jailer := guardian.NewGenericJailer(redisConfStore, logger.WithField("context", "route-jailer"), redisCounter, redisConfStore, reporter)
 	ipRateLimiter := guardian.NewIPRateLimiter(redisConfStore, logger.WithField("context", "ip-rate-limiter"), reporter, redisCounter)
 	routeRateLimiter := guardian.NewRouteRateLimiter(redisConfStore, logger.WithField("context", "route-rate-limiter"), reporter, redisCounter)
-	condFuncChain := guardian.DefaultCondChain(whitelister, blacklister, jailer, ipRateLimiter, routeRateLimiter)
+	condFuncChain := guardian.DefaultCondChain(whitelister, blacklister, jailer, routeRateLimiter, ipRateLimiter)
```

Comment:
This order change allows us to prioritize the route rate limit and stop processing the request if an enabled rate limit for the route is found.